run-integration-tests.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+set +eux pipefail
+# Usage: After running docker compose up, execute this script to check that each 
+# cli command succeeds. The CLI will make requests to the server.
+pushd cli
```

Comment:
Guessing this script only works from the root? 

I think you might want to remove this script or maybe put everything into `cargo test` or `just` or `make` file, whichever the team prefers

Reply:
I have another branch trying to do the proper cargo test route but it was taking too long so this is a stop gap to just run ourselves as a spot check.

Reply:
Had issues with docker compose up exiting in the test prematurely